lang/python/python-service-identity/Makefile

```diff
@@ -9,11 +9,12 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-service-identity
 PKG_VERSION:=18.1.0
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```

Comment:
this change looks cosmetic;
i don't think we need to bump PKG_RELEASE;
typically that's worth doing if changing something related to build [i.e. CFLAGS, LDFLAGS, patches being added/removed]; anything that changes the binary output 

[or at least that's my understanding]

Reply:
Will amend - thanks!